src/IO/lbsLoadReaderVT.py

```diff
@@ -169,13 +175,22 @@ def read(self, node_id, time_step=-1, comm=False):
                             print(bcolors.HEADER
                                 + "[LoadReaderVT] "
                                 + bcolors.END
-                                + "iteration = {}, object id = {}, time = {}".format(
+                                + "iteration = {}, object id = {}, time = {}, subphases = {}".format(
                                 phase,
                                 o_id,
-                                time))
+                                time,
+                                Nsubphases,
+                                end=''))
+                                
+                            for i in range(0,Nsubphases):
+                                print("subphase-time-" + str(i) + "= {}".format(
+                                    row[4 + i],
+                                end=''))
+                                
+                            print()
 
                 # Handle four-entry case that corresponds to a communication weight
-                elif n_entries == 5:
+                elif '[' not in row[4]:
```

Comment:
The `if` and `elif` cover all possibilities. This `elif` should keep itself more narrowly tailored to the case it actually handles, so that the `else` still gets run for lines that aren't ever going to be parsed correctly

Reply:
My latest commit addresses this issue.

Reply:
@PhilMiller 
https://github.com/DARMA-tasking/LB-analysis-framework/blob/30477157960faf905b351d385652fe17ae424c28/src/IO/lbsLoadReaderVT.py#L194